tensorflow_datasets/core/dataset_info.py

```diff
@@ -265,6 +268,29 @@ def _dataset_info_path(self, dataset_info_dir):
   def _license_path(self, dataset_info_dir):
     return os.path.join(dataset_info_dir, LICENSE_FILENAME)
 
+  def visualize_dataset(self):
+    return self._visualize_dataset(self._builder, 'train')
+
+  def _visualize_dataset(self, builder, split):
```

Comment:
Please merge with above function.
Or is there a reason to have both visualize_dataset and _visualize_dataset functions?

Reply:
No, just a little mistake. I think I was confused when I writing this.